android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/PdfDisplayDemoView.java

```diff
@@ -0,0 +1,72 @@
+package com.liferay.mobile.formsscreenletdemo.view;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+import com.liferay.mobile.screens.viewsets.defaultviews.dlfile.display.PdfDisplayView;
+
+/**
+ * @author Luísa Lima
+ */
+public class PdfDisplayDemoView extends PdfDisplayView {
+
+	private Button nextPage;
+	private Button previousPage;
+	private Button goToPageButton;
+	private EditText goToPage;
+	private LinearLayout linearLayoutButtons;
+	private ImageView imagePdf;
+	private ProgressBar progressBarHorizontal;
+	private TextView progressText;
+	private TextView title;
+
+	public PdfDisplayDemoView(Context context) {
+		super(context);
+	}
+
+	public PdfDisplayDemoView(Context context, AttributeSet attrs) {
+		super(context, attrs);
+	}
+
+	public PdfDisplayDemoView(Context context, AttributeSet attrs, int defStyleAttr) {
+		super(context, attrs, defStyleAttr);
+	}
+
+	@Override
+	protected void onFinishInflate() {
+		super.onFinishInflate();
+
+		imagePdf = findViewById(com.liferay.mobile.screens.R.id.liferay_pdf_renderer);
```

Comment:
You are only using `com.liferay.mobile.screens.R` in the whole file, instead of use the full path name here import it and just call `R.id.liferay_pdf_renderer`

Reply:
Okay